lib/off_broadway/kafka/showtime_handler.ex

```diff
@@ -25,22 +25,21 @@ defmodule OffBroadway.Kafka.ShowtimeHandler do
   def init(args) do
     broadway_module = Keyword.fetch!(args, :broadway_module)
     opts = Keyword.get(args, :opts, [])
+    producer_selector = Keyword.get(args, :producer_selector, &default_selector/1)
```

Comment:
I am not sure I understand the use case where a user would want to control which producer is selected.

Reply:
I can’t think of one personally either beyond arbitrarily selecting the first one or a random one didn’t feel quite right as the only available option. I’m not married to it though.

Reply:
it doesn't hurt anything, my only concern is adding confusion for a user when reading the documentation.  Other than that, I am good.

Reply:
agreed; i don't want to include unnecessary confusion or complexity. it's definitely cleaner just making the default function the only supported behavior.